docs/run-keep-ecdsa.adoc

```diff
@@ -0,0 +1,419 @@
+    "0xDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD",
+    "0xEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE"
+  ]
+
+[Storage]
+  DataDir = "/my/secure/location"
+
+[LibP2P]
+  Peers = ["/ip4/127.0.0.1/tcp/3919/ipfs/njOXcNpVTweO3fmX72OTgDX9lfb1AYiiq4BN6Da1tFy9nT3sRT2h1"]
+  Port = 3919
+  # Uncomment to override the node's default addresses announced in the network
+  AnnouncedAddresses = ["/dns4/example.com/tcp/3919", "/ip4/80.70.60.50/tcp/3919"]
+
+[TSS]
+# Timeout for TSS protocol pre-parameters generation. The value
+# should be provided based on resources available on the machine running the client.
+# This is an optional parameter, if not provided timeout for TSS protocol
+# pre-parameters generation will be set to `2 minutes`.
+  PreParamsGenerationTimeout = "2m30s"
+
+```
+
+==== Parameters
+
+[%header,cols=4*]
+|===
+|`ethereum`
+|Description
+|Default
+|Required
+
+|`URL`
+|The Ethereum host your keep-ecdsa will connect to.  Websocket protocol/port.
+|""
+|Yes
+
+|`URLRPC`
+|The Ethereum host your keep-ecdsa will connect to.  RPC protocol/port.
+|""
+|Yes
+|===
+
+[%header,cols=4*]
+|===
+|`ethereum.account`
+|Description
+|Default
+|Required
+
+|`KeyFile`
+|The local filesystem path to your Keep operator Ethereum account keyfile.
+|""
+|Yes
+|===
+
+[%header,cols=4*]
+|===
+|`ethereum.ContractAddresses`
+|Description
+|Default
+|Required
+
+|`BondedECDSAKeepFactory`
+|Hex-encoded address of the BondedECDSAKeepFactory Contract.
+|""
+|Yes
+|===
+
+[%header,cols=4*]
+|===
+|`SanctionedApplications`
+|Description
+|Default
+|Required
+
+|`Addresses`
+|Comma delimited hex-encoded list of application addresses authorized for a given operator.
+|[""]
+|Yes
+|===
+
+[%header,cols=4*]
+|===
+|`LibP2P`
+|Description
+|Default
+|Required
+
+|`Peers`
+|Comma separated list of network peers to boostrap against.
+|[""]
+|Yes
+
+|`Port`
+|The port to run your instance of Keep on.
+|3919
+|Yes
+
+|`AnnouncedAddresses`
+|Multiaddr formatted hostnames or addresses annouced to the
+Keep Network. More on multiaddr format
+https://docs.libp2p.io/reference/glossary/#multiaddr[in the libp2p
+reference].
+|[""]
+|No
+|===
+
+[%header,cols=4*]
+|===
+|`Storage`
+|Description
+|Default
+|Required
+
+|`DataDir`
+|Location to store the Keep nodes group membership details.
+|""
+|Yes
+|===
+
+[%header,cols=4*]
+|===
+|`TSS`
+|Description
+|Default
+|Required
+
+|`PreParamsGenerationTimeout`
+|Timeout for TSS protocol pre-parameters generation.
+|"2m"
+|No
+|===
+
+== Build from Source
+
+See the https://github.com/keep-network/keep-core/tree/master/docs/development#building[building] section in our developer docs.
+
+== Docker
+
+=== Get Image
+
+Doesn't exist for the public yet.
+
+=== Run Image
+This is a sample run command for illustration purposes only.
+
+```
+export KEEP_CLIENT_ETHEREUM_PASSWORD=$(cat .secrets/eth-account-password.txt)
+export KEEP_CLIENT_CONFIG_DIR=$(pwd)/config
+export KEEP_CLIENT_PERSISTENCE_DIR=$(pwd)/persistence
+
+docker run -dit \
```

Comment:
There is no reason to use the `-it`  options here if using `-d`.  You are essentially telling Docker to open an interactive terminal but detach from it.  
```suggestion
docker run -d \
```

Reply:
> There is no reason to use the `-it` options here if using `-d`.

This is a weird one.  Even in detached mode if you set the `entrypoint` to `bash` or `sh` you need the pseudo-terminal to keep the container running in detached mode, hence the `-dit`.  To your point this is kind of a holdover from `docker run` scripts I use for local testing, where I frequently `--entrypoint /bin/sh`.  I should be able to remove this here.

Reply:
Gotcha, that makes sense.  I'm just being picky here really.